### PR TITLE
Content

### PR DESCRIPTION
**Pull request by evadb**
This PR adds in all the new content in English & French, hooray 🎉 !

I am tagging everyone in this PR because @sidewalkballet and @samsadasivan I would love a careful review of the content, and @buckley-w-david I'd love a look over of the code.

### CODE REVIEW

File: track/templates/en/layout-en.html [hunk 1]
```diff
@@ -51,7 +51,7 @@ <h2 class="py-6 text-lg sm:text-xl hidden sm:block"><a class="text-grey-darkest
 		                    <a class="text-https-blue no-underline hover:underline" href="/en/organizations/">Check Compliance</a>
```
Comment (sidewalkballet) [inline track/templates/en/layout-en.html]: downcase compliance for consistency

Comment (evadb) [reply track/templates/en/layout-en.html]: 👍

File: track/templates/includes/fr/modal.html [hunk 2]
```diff
@@ -5,40 +5,59 @@
...
+		    </ul>
 
-			<hr class="border mt-4 mb-4">
+			<hr class="border mt-6 mb-6">
 
-			<h3 class="text-xl lg:text-2xl font-bold mb-4">Enforces HTTPS</h3>
-		    <p class="text-base lg:text-lg">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla suscipit dui nec consequat bibendum. Nam condimentum ac turpis non vestibulum.</p>
-		    <p class="text-base lg:text-lg">Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Aliquam id lacus ac erat congue placerat.</p>
-		    <p class="text-base lg:text-lg"><a href="" class="text-https-blue hover:text-https-blue-darker font-bold">Learn more</a></p>
+			<h3 class="text-xl lg:text-2xl font-bold mb-4">Utilise le protocole HTTPS</h3>
```
Comment (sidewalkballet) [inline track/templates/includes/fr/modal.html]: The table says "applique le protocole HTTPS" - should be consistent one way or another

File: track/templates/en/layout-en.html (hunk 1 above)
Comment (sidewalkballet) [inline track/templates/en/layout-en.html]: downcase compliance for consistency (on line 67 as well)

File: track/templates/includes/fr/header.html [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<div class="container mx-auto items-center sm:w-4/5 xl:w-3/5 mt-6">
+	<h1 class="text-4xl sm:text-5xl">Vérifier la conformité</h1>
+	<h2 class="mb-4 sm:mb-6 text-xl sm:text-2xl">Dernière mise à jour {{ scan_date | display_date("fr") }}</h2>
+	<p class="text-lg mb-6">Ce tableau indique dans quelles proportions les sites et services en ligne gouvernementaux respectent les bonnes pratiques en matière de sécurité Web décrites dans l’Avis de mise en œuvre de la Politique sur la technologie de l’information (AMPTI) : Utiliser le protocole HTTPS pour des connexions Web sécurisées.</p>
```
Comment (sidewalkballet) [inline track/templates/includes/fr/header.html]: French ITPIN name is `Mise en œuvre de HTTPS pour les connexions Web sécurisées`

File: track/templates/fr/help.html [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
 {% block pageid_fr %}aide{% endblock %}
 
 {% block content %}
 <section id="main-content" class="flex-1">
 	<div class="container mx-auto items-center sm:w-4/5 xl:w-3/5 mt-6">
-		<h1>Rétroaction</h1>
-		<p>Veuillez transmettre vos questions ou commentaires au sujet de ce tableau de bord ou de la mise en œuvre de HTTPS par courriel à la <a href="mailto:zzTBSCybers@tbs-sct.gc.ca" class="text-https-blue hover:text-black font-bold">Cybersécurité du SCT</a> ou joindre le groupe <a href="https://gcconnex.gc.ca/groups/profile/35218846/gc-https-everywhere-2018-https-partout-dans-le-gc-2018?language=en" class="text-https-blue hover:text-black font-bold">« HTTPS Everywhere » sur GCconnex</a>.</p>
+		<h1>Obtenir de l’aide</h1>
+		<h2 class="font-bold mb-4">Grand public</h2>
+		<p>Pour des explications concernant tout aspect de l’application de l’Avis de mise en œuvre de la Politique sur la technologie de l’information (AMPTI)&nbsp;: Utiliser le protocole HTTPS pour les connexions Web sécurisées, veuillez communiquer avec le Secrétariat du Conseil du Trésor du Canada par l’entremise des Renseignements au public.</p>
```
Comment (sidewalkballet) [inline track/templates/fr/help.html]: French name is `Mise en œuvre de HTTPS pour les connexions Web sécurisées`

File: track/templates/en/help.html [hunk 5]
```diff
@@ -1,14 +1,19 @@
...
+		<h1>Get help</h1>
+		<h2 class="font-bold mb-4">General public</h2>
+		<p>For interpretation of any aspect of Implementing HTTPS for Secure Web Connections:   Information Technology Policy Implementation Notice (ITPIN), contact Treasury Board of Canada Secretariat through Public Enquiries.</p>
+		<h2 class="font-bold mt-4 mb-4">Government of Canada employees</h2>
+		<p>Individuals at departments should contact their departmental information technology group for any questions regarding this ITPIN. Individuals can also join the <a href="https://gcconnex.gc.ca/groups/profile/35218846/gc-https-everywhere-2018-https-partout-dans-le-gc-2018?language=en" class="text-https-blue hover:text-black font-bold">HTTPS Everywhere group on GCconnex</a>, or the #HTTPSEverywhere channel on GCcollab Message.</p>
+		<p>Individuals from a departmental information technology group may contact the <a href="mailto:zzTBSCybers@tbs-sct.gc.ca" class="text-https-blue hover:text-black font-bold">TBS Cyber Security mailbox</a> for interpretations of this ITPIN.</p>
```
Comment (sidewalkballet) [inline track/templates/en/help.html]: whether or not `mailbox` is included in the `<a>` tag should be consistent with line 16

File: track/templates/en/help.html [hunk 6]
```diff
@@ -1,14 +1,19 @@
...
 {% block pageid_fr %}aide{% endblock %}
 
 {% block content %}
 <section id="main-content" class="flex-1">
 	<div class="container mx-auto items-center sm:w-4/5 xl:w-3/5 mt-6">
-		<h1>Feedback</h1>
-		<p>For inquiries, questions, or comments about this dashboard or implementing HTTPS, please send an email to <a href="mailto:zzTBSCybers@tbs-sct.gc.ca" class="text-https-blue hover:text-black font-bold">TBS Cyber Security</a> or join the <a href="https://gcconnex.gc.ca/groups/profile/35218846/gc-https-everywhere-2018-https-partout-dans-le-gc-2018?language=en" class="text-https-blue hover:text-black font-bold">HTTPS Everywhere group on GCconnex</a>.</p>
+		<h1>Get help</h1>
+		<h2 class="font-bold mb-4">General public</h2>
+		<p>For interpretation of any aspect of Implementing HTTPS for Secure Web Connections:   Information Technology Policy Implementation Notice (ITPIN), contact Treasury Board of Canada Secretariat through Public Enquiries.</p>
```
Comment (sidewalkballet) [inline track/templates/en/help.html]: We should be using `<abbr>`s on ITPIN (and everywhere else it's used)

File: track/templates/en/help.html [hunk 7]
```diff
@@ -1,14 +1,19 @@
...
+		<h2 class="font-bold mt-4 mb-4">Government of Canada employees</h2>
+		<p>Individuals at departments should contact their departmental information technology group for any questions regarding this ITPIN. Individuals can also join the <a href="https://gcconnex.gc.ca/groups/profile/35218846/gc-https-everywhere-2018-https-partout-dans-le-gc-2018?language=en" class="text-https-blue hover:text-black font-bold">HTTPS Everywhere group on GCconnex</a>, or the #HTTPSEverywhere channel on GCcollab Message.</p>
```
Comment (sidewalkballet) [inline track/templates/en/help.html]: GCcollab Message can link to https://message.gccollab.ca/ for both EN and FR

**Pull request merged**